MultiOracle.md

```diff
@@ -0,0 +1,462 @@
+This process results in 1 or 2 secondary oracle digit prefixes for every digit prefix of the primary oracle (see [algorithm below](#algorithm)). 
+
+### 2-of-2 Oracles with Bounded Error
+
+In the two oracle case, mark one oracle as primary and the other as secondary.
+First, generate all digit prefixes for the primary oracle as is done in the single-oracle
+case for [numeric outcome DLCs](NumericOutcome.md).
+Then we shall construct a list of 1 or 2 secondary digit prefixes whose corresponding adaptor points will
+be combined with the adaptor point corresponding to the primary oracle digit prefix.
+This scheme can be thought of as computing a set of 1 or 2 digit prefixes for the secondary oracle to
+cover at least the minimum range and at most the maximum range of outcomes allowed given each
+primary oracle's digit prefix.
+
+The maximum possible support range is parameterized by a maximum error exponent, `maxErrorExp`,
+where all difference more than `maxError = 2^maxErrorExp` are guaranteed **not** to be supported.
+
+The minimum supported range is parameterized by a minimum support exponent, `minSupportExp`, where
+all differences of up to `minSupport = 2^minSupportExp` are guaranteed to be supported.
+It is required that `minSupportExp` be strictly less than `maxErrorExp` because this gap is key to the 
+functioning of the following algorithm.
+
+If the two oracles sign outcomes that are between `minSupport` and `maxError` apart, then this procedure
+can provide only probabilistic guarantees.
+That said, DLC participants can decide in advance whether they wish to maximize the probability of
+failure or of success in these cases.
+
+With these parameters in mind, we shall now detail the algorithm for computing the secondary oracle's
+digit prefixes given those of the first.
+
+#### Algorithm
+
+First, let us define some subprocedures.
+
+* `computePrefixIntervalBinary` takes as input a digit prefix corresponding to an outcome interval, and the
+  total number of binary digits to be signed by an oracle, and returns an interval `[start, end]`
+  which that digit prefix covers.
+* `numToVec` takes as input an integer representing the left endpoint of a digit prefix's range, the number
+  of binary digits to be signed by the oracle, and a number of digits to ignore/omit, and returns
+  the binary digits corresponding to that digit prefix.
+
+We take as input the number of digits to be signed by the oracle, `numDigits`, the binary digits of the primary
+oracle's digit prefix which we wish to cover with the secondary oracle, `primaryDigits`, `maxErrorExp`, `minSupportExp`,
+and a boolean flag `maximizeCoverage` which signals whether to maximize the probability of success for
+outcomes differing by less than `maxError` and more than `minSupport` or not (in which case failure is maximized).
+
+Let `[start, end] = computePrefixIntervalBinary(primaryDigits, numDigits)`,
```

Comment:
Perhaps provide an example of `primaryDigits`

Reply:
I think what might cover this without requiring an abrupt change of direction is a link to the reference implementation (which will be added before merging this PR) and may a "see examples here" with a link to test vectors which will be example inputs to `computePrefixIntervalBinary` with their corresponding outputs.

Thoughts?

Reply:
I think that would do it. Perhaps there should also be a link to the [CET Compression concrete example](https://github.com/discreetlogcontracts/dlcspecs/blob/898e81a23c30ac4bc66209b20697e76e4a1da7bd/CETCompression.md#concrete-example).